github_action_build.sh

```diff
@@ -15,7 +15,6 @@ echo "branch name: ${BRANCH_NAME}"
 if [ -z "${GITHUB_HEAD_REF}" ]; then
     cmake -DCMAKE_BUILD_TYPE=Debug -DCODE_COVERAGE=1 -DENABLE_RAFT_STATS=1 ../
     make -j2 raft_cov
-    bash <(curl -s https://codecov.io/bash) -f raft_cov.info.cleaned
```

Comment:
Hi @lucasgonze , thanks for submitting this PR.

I understand all the other changes you made, but I don't get how things work without this line; as `codecov-action` does not know what the result file is. 

This project uses `lcov`, it generates `raft_cov.info` and then `raft_cov.info.cleaned` after excluding given paths. Shouldn't we pass some parameters to `codecov-action`?

Reply:
Maybe I should test it in other temporary repository.